enclave.go

```diff
@@ -304,10 +310,12 @@ func (e *Enclave) setupAcme() error {
 
 	elog.Printf("ACME hostname set to %s.", e.cfg.FQDN)
 	var cache autocert.Cache
-	if err = os.MkdirAll(acmeCertCacheDir, 0700); err != nil {
-		return fmt.Errorf("Failed to create cache directory: %w", err)
```

Comment:
Is it not still helpful to create the cache directory if necessary?

Reply:
I noticed that this code is redundant. The [cache implementation](https://pkg.go.dev/golang.org/x/crypto/acme/autocert#DirCache) that we're using creates the directory if it doesn't exist yet.